superset-frontend/src/views/CRUD/dashboard/DashboardList.tsx

```diff
@@ -69,6 +69,7 @@ interface Dashboard {
   dashboard_title: string;
   id: number;
   published: boolean;
+  favorite: boolean;
```

Comment:
is this actually part of the dashboard api response?

Reply:
No. Thanks for noticing.